src/main/java/com/zzpj/dtos/RoleDto.java

```diff
@@ -0,0 +1,17 @@
+package com.zzpj.dtos;
+
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@NoArgsConstructor
+@Data
+@JsonPropertyOrder({"name", "version"})
```

Comment:
jeszcze id tutaj

Reply:
tu mam pytanie, czemu akurat w RoleDto ma byc tutaj to ID, a w niektórych dto nie ma? chciałabym to zrozumieć a nie poprawiać żeby było poprawione ;)